fix the topbar position and its background

Mostafa-alsaygh opened the pull request:
# What does this PR do?

The TopBar in the ReviewsScreen has been moved from being an overlay
within a Box to the top of a Column layout and remove the static padding. This change ensures the
TopBar is consistently positioned at the top of the screen, and the
LazyColumn content appears below it.

The title property was also removed from the TopBar as it was already
being displayed by the EmptyReviews composable when no reviews are present.

Mostafa-alsaygh committed:
refactor: Adjust TopBar position in ReviewsScreen

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

BassantM reviewed (dismissed):
it would be at the center  @Mostafa-alsaygh 
<img width="447" height="791" alt="image" src="https://github.com/user-attachments/assets/e99df47f-3c28-4f3c-8e4a-036590f68a1d" />

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

Mostafa-alsaygh committed:
Refactor: Center EmptyReviews component in ReviewsScreen

Mostafa-alsaygh committed:
Fix: Add title string resource to ReviewsScreen TopBar

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

london-bot[bot] committed:
Branch was auto-updated.

yusufnasserdev committed:
fix: update no reviews message

Removes unnecessary quotes from the "no reviews" string and adds an exclamation mark.

Mostafa-alsaygh committed:
Merge branch 'develop' into fix/reviews-screen-top-bar

london-bot[bot] committed:
Branch was auto-updated.

Mostafa-alsaygh committed:
Merge branch 'develop' into fix/reviews-screen-top-bar

# Conflicts:
#	presentation/src/main/java/com/london/presentation/feature/reviews/reviewsScreen.kt

Mostafa-alsaygh committed:
Refactor: Update ReviewsScreen layout

The ReviewsScreen now uses a Column to structure its content.
The TopBar has been integrated directly into the Column.
The EmptyReviews component is now centered within a Box that takes up remaining space if the reviews list is empty.
